CMake/cdat_modules_extra/cdat_common_environment.cmake.in

```diff
@@ -1,5 +1,5 @@
 message("[INFO] ADDITIONAL CFLAGS ${ADDITIONAL_CFLAGS}")
-set(ENV{PATH} "@cdat_EXTERNALS@/bin:$ENV{PATH}")
+set(ENV{PATH} "@SB_BIN_DIR@:@cdat_EXTERNALS@/bin:$ENV{PATH}")
```

Comment:
@chaosphere2112 I would suggest we remove it since it seems like its not really helping with the issue.